docker-compose.yml

```diff
@@ -11,11 +11,14 @@ services:
     volumes:
       - api-postgres:/var/lib/postgresql/data
 
-  thumbs:
-    image: willnorris/imageproxy
+  thumbnails:
+    image: h2non/imaginary:latest
```

Comment:
Should we pin this to a specific version, like we do the rest of our dependency images? I know the original one wasn't but it seems wrong not to.

Reply:
I've been waiting on a response for https://github.com/h2non/imaginary/issues/390 to resolve this.

Reply:
Wow, the last release on GitHub was also 2 years ago :confused:

Reply:
I guess the worst case here is we somehow build it ourselves in a fork?

Reply:
Attempted this in PR #641.